src/watchlists/WatchList.jsx

```diff
@@ -30,15 +41,25 @@ export default function watchlist({ item }) {
           />
         </TouchableHighlight>
       </View>
-      {isOpen ? item.stocks.map((stock) => (
-        <WatchListItem
-          key={stock.id}
-          ticker={stock.ticker}
-          currPrice={stock.currPrice}
-          dailyChange={stock.dailyChange}
-          volumn={stock.volumn}
-        />
-      )) : null}
+      {isOpen && (
+        <>
+          {item.stocks.map((stock, i) => (
+            <WatchListItem
+              key={stock.id}
+              isEdit={isEdit}
```

Comment:
create a separated component for editable watchlist items, pay attention to the single responsibility. Components with more funcionlaity are much harder to test